### PR TITLE
Added the OpenHack link to the README

### PR DESCRIPTION
**Pull request by michelle123lam**
First pass at #1!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-## [OpenHack]: Your first PR!
+## [OpenHack](http://openhack.info): Your first PR!
```
Comment (bclyang) [inline README.md]: We should link to the HTTPS version!

**Comment by bclyang** [comment]
Hey, reviewed your changes. Looks good-- just change that one thing!

**Comment by michelle123lam** [comment]
Ok, just made the change!

**Comment by bclyang** [comment]
LGTM

**Pull request merged**